aiohttp/connector.py

```diff
@@ -411,8 +411,8 @@ class TCPConnector(BaseConnector):
     """
 
     def __init__(self, *, verify_ssl=True, fingerprint=None,
-                 resolve=_marker, use_dns_cache=_marker,
-                 family=0, ssl_context=None, local_addr=None,
+                 resolve=_marker, use_dns_cache=_marker, family=0,
+                 ssl_context=None, local_addr=None, tcp_timeout=None,
```

Comment:
May be just `timeout`? Unlikely `TCPConnector` will get `udp_timeout` or `sctp` one.